index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Akhror Valiev</title>
+
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1" />
+
+	<link href="https://fonts.googleapis.com/css2?family=Russo+One&display=swap" rel="stylesheet">
+
+	<link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&family=Russo+One&display=swap" rel="stylesheet">
+
+	<link rel="stylesheet" type="text/css" href="default.css">
+
+</head>
+<body>
+
+	<section class="s1">
+		<div class="main-container">
+			<div class="greeting-wrapper">
+				<h1>Hi, I'm Akhror Valiev</h1>
+			</div>
```

Comment:
here, you should use a header tag instead of the div. Because a header tag is a semantic HTML tag that improves website accessibility and SEO of the page.

Reply:
Great work @akhror-valiev. You made a very good review here with excellent explanation. Also kuddos on using the github review feature. I suggest you make a change however to how your comments are phrased:

- [ ] Currently your comments feel like orders:

This is an example of how comments can be phrased:
* This is a polite suggestion: *"I think it would be better to use semantic tags"*
This is a requirement for you to successfully get approved:
<img width="1660" alt="Screenshot 2022-09-26 at 12 59 35" src="https://user-images.githubusercontent.com/48546992/192269613-7a1f4f92-5375-4ffe-b461-9df4fd5fd421.png">